sunpy/io/tests/test_fits.py

```diff
@@ -176,3 +177,11 @@ def test_warn_longkey():
                                'goodkey': 'test'})
     assert 'GOODKEY' in fits.keys()
     assert 'BADLONGKEY' not in fits.keys()
+
+
+def test_read_memmap():
+    # Test that memmap is passed correctly to the FITS reader
+    # Expecting ValueError: not enough values to unpack (expected 2, got 1)
```

Comment:
The ValueError should be due to a bug in the test function itself, as using memmap is not expected to produce an error.

Reply:
We need 2 parameters in the `sunpy.io._fits.read()` function,

filepath : `str`
The fits file to be read.

I passed `AIA_193_JP2`.

Can you help me, I don't know how to get the second parameter which is:

hdus : `int` or iterable
The HDU indexes to read from the file.

Reply:
The second parameter is optional. (It's for selecting a specific array of data inside the FITS file if it contains multiple but you only need some of them.) The unpack error is occurring when Python tries to unpack the output of the read function in the test: e.g. `a, b = [1, 2]` works but `a, b = [[1, 2]]` produces an error because the outer object has length 1.

Reply:
Done!
Do the assertion test looks good?